spec/lib/irs1040_pdf_spec.rb

```diff
@@ -8,26 +8,6 @@
   let(:tax_year) { 2021 }
   let(:submission) { create :efile_submission, :ctc, tax_year: tax_year }
 
-  describe "initialization" do
```

Comment:
i feel like this test ensured that there was a fallback to the intake address when there was not a persisted verified address object to refer to and I dont know that that behavior is otherwise tested in the spec.

Reply:
yup we're going to add another way of testing this